crates/base/src/worker_ctx.rs

```diff
@@ -123,6 +133,24 @@ pub async fn create_worker(
     }
 }
 
+async fn send_user_worker_request(
+    worker_channel: mpsc::UnboundedSender<WorkerRequestMsg>,
```

Comment:
This function will be called inside a tokio async task. Since async tasks require to be sent between threads, this cannot be passed as a reference. Also, `clone` on channels are quite cheap (they are just  `Rc<RefCell>` under the hood)